src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -49,6 +49,7 @@ See the LICENSE file in the project root for more information.
     <LinkerArg Include="@(NativeLibrary)" />
     <LinkerArg Include="@(AdditionalNativeLibrary)" />
     <LinkerArg Include="-g" />
+    <LinkerArg Include="-Wl,-rpath,'$ORIGIN'" />
```

Comment:
@sergiy-k this syntax doesn't look right. I believe it should be 
`<LinkerArg Include="-Wl,-rpath='$ORIGIN'" />`